cpp/dolfinx/fem/Form.h

```diff
@@ -86,6 +95,9 @@ struct integral_data
 
   /// The entities to integrate over
   std::vector<std::int32_t> entities;
+
+  /// Indicator of which coefficients (from the form) that is in this integral
```

Comment:
See above question on what  enabled_coefficients is.

Reply:
Enabled coefficients is a list of booleans, corresponding to the N coefficients in the form. 
As each form is split into M integrals, not every integral uses every coefficient.